index.html

```diff
@@ -11,19 +11,35 @@
     <header>
     <nav class="main-container">
       <a href="#" class="main-navbar-logo">John Doe</a>
-      <a href="#" class="main-navbar-menu">
+      <a href="#" class="main-navbar-menu logo-right-whole">
         <img src="./images/Union.png" alt="Menu"/>
+        <div class="righthand-of-logo righthand-of-logo-2">
+          <h3 class="porfolio-logo logo-portfolio-right-content">
+              Portfolio
+          </h3>
+          <h3 class="about-about logo-portfolio-right-content">
+              About
+          </h3>
+          <h3 class="contact-logo logo-portfolio-right-content">
+              Contact
+          </h3>
+          <img src="./images/Union2.svg" alt="Envelop Icon">
+       </div>
       </a>
     </nav>
     </header>
     <section class="section-header-box">
       <section class="section-header-content">
-        <h1 class="section-header-title">Hey There. I'm Olamarx</h1>
+        <div class="section-header-content-except-icons">
+          <h1 class="section-header-title">Hey There. I'm Olamarx</h1>
         <h2 class="section-header-subtitle">I’m a Software Developer</h2>
         <p class="section-header-text-description">
-          I can help you build a product , feature or website Look through some of my work and experience! If you like
+          I can help you build a product, feature or website Look through some of my work and experience! If you like
           what you see and have a project you need coded, don’t hestiate to contact me.
         </p>
+        </div>
+
+
         <ul class="social-media-link">
           <li>
             <a href="#" class="social-media icon">
```

Comment:
- [ ] Please add the right font family and font-weight to all heading text and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/159834192-8acefb6b-b80b-41a4-b94b-dde166034916.png) | ![image](https://user-images.githubusercontent.com/68796232/159834145-3d912093-2d8d-4a39-b61b-1eaa1a073f27.png)  |

Reply:
I added the font family and the font-weight to all the heading texts as provided in Figma. I mean the name of the font family and a fallback general name too.
You can confirm in the CSS with the class name section-header-title.

![done](https://user-images.githubusercontent.com/86200356/159924653-fc1da171-2f0a-4b8a-be2b-6066e2b89476.jpg)